README.md

```diff
@@ -1,24 +1,218 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
 
-Things you may want to cover:
+Modify this file to match your project and remove sections that don't apply.
 
-* Ruby version
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
 
-* System dependencies
+OPTIONAL SECTIONS:
+- FAQ
 
-* Configuration
+After you're finished please remove all the comments and instructions!
+-->
 
-* Database creation
 
-* Database initialization
+<!-- TABLE OF CONTENTS -->
 
-* How to run the test suite
+# 📗 Table of Contents
 
-* Services (job queues, cache servers, search engines, etc.)
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Deployment instructions
+<!-- PROJECT DESCRIPTION -->
 
-* ...
+# 📖 [DavRich Recipe App] <a name="about-project"></a>
+
+
+
+**[Budget App]** is a full stack web app with which a user can add basic budget items and monitor their spendings, the application allows users to enter items and amounts to be spent on every item
+
+## 🛠 Built With <a name="built-with">Ruby on Rails</a>
+
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Authentication with devise**
+- **Nested routing**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](https://budget-app-3c3n.onrender.com)
+- [Video](https://drive.google.com/file/d/19fGLUPm0SxLqDOKr1xkhpyk1G5hcQRDh/view?usp=sharing)
+
```

Comment:
Thank you for your effort to make a clear video. Please, let's add other project requirements and make it more attractive for viewers or recruiters.
- [ ] The video should contain a highlight about a piece of the code the student finds interesting or is proud of making. this requires moving into your VS code and show a specific part of code. Kindly work on this.

Reply:
Thank you for your suggestions, i have recorded a new video highlighting my project (https://drive.google.com/file/d/16oDY-3v6GjoWdDSYWB2Khpmdg_u8z9xG/view?usp=sharing) and i have updated the link in my readme file